spec/factories/evaluation_criteria.rb

```diff
@@ -0,0 +1,28 @@
+FactoryBot.define do
+  factory :evaluation_criterion, class: 'EvaluationCriterion' do
+    # Associations
+    association :evaluation_form
+
+    # Fields
+    title { "Criterion #{Faker::Lorem.word}" }
+    description { Faker::Lorem.sentence }
+    points_or_weight { rand(0..100) }
+    scoring_type { :numeric }
+    option_range_start { 0 }
+    option_range_end { 4 }
+    option_labels { [] }
```

Comment:
these can be nil for scoring type :numeric

Reply:
The labels? Yeah I need to think of how these will actually be structured. Are these only for `binary` scale? In which case I assume the `option_range_start` and `option_range_end` would be `nil`?

I'm assuming it'd be something like this for example. `[{"disagree": 0, "slightly disagree": 1, .... "agree": 4}]`

The validations on binary scale could get tricky especially taking scoring into account if they aren't validated to be specifically a range under the hood

Reply:
oh sorry, labels can probably default to empty list [] just so it's valid json but i meant more the range_start and range_end. labels only apply for rating and binary scale but it can be empty for numeric scale. 👌

Reply:
for binary scale scoring, I know that it's all or nothing scoring, so the submission either gets the full points for that criteria or 0 depending on the judges choice on the binary score.

Reply:
Oh that kind of binary. So does this often end up being a boolean type of yes/no? Are there cases where they'd label them as something else?

Reply:
I see they can add a 0 and 1 label. So that's fine

Reply:
yes binary scale has customizable labels